docs/components/search/index.html

```diff
@@ -0,0 +1,39 @@
+--- 
+  title: Search 
```

Comment:
remove indentation before `title: Search` and pls add also: elements/hx-search: <hx-search> underneath the `title: Search`